.eslintrc.cjs

```diff
@@ -1,3 +1,6 @@
-module.exports = {
-  ...require('@linzjs/style/.eslintrc.js'),
-};
+const cfg = require('@linzjs/style/.eslintrc.js');
+
+cfg.parserOptions.ecmaVersion = 2020;
+
+module.exports = cfg;
```

Comment:
@blacha am I right in saying this is config for linting/formatting .js files?

Reply:
This is because the `@linzjs/style` currently defaults to commonJS modules or ecma version < 2015.

all new pacakges inside of LINZ are being moved to ecma version 2020, `@linzjs/style` has not been updated yet for this.

Reply:
It doesn't look relevant for the PR, though, or am I misunderstanding something?

Reply:
It is relevant,

Since this is the first piece of javascript in this repo, and this repo is using ecma modules we needed to upgrade the linter to use the ecma module parser.

without this the linter would not function.